src/renderer/components/Sidebar.tsx

```diff
@@ -181,7 +181,10 @@ const Sidebar = React.memo(
           </div>
           <div key='logout' className='sidebar-item' onClick={onLogout}>
             {tx('switch_account')}
-            <OtherAccountsUnreadBadge top='-38px' left='-14px' />
+            <OtherAccountsUnreadBadge
+              style={{ top: '-38px', left: '110px' }}
```

Comment:
I would prefer having these offsets in they (s)css unless the are either for prototyping or dynamic.

Reply:
The thing is that the parent component has to determine the offsets according to its use. The offsets are different for the hamburger menu icon and in the sidebar.